samples_project/Assets/SampleViewer/Samples/FeatureLayer/FeatureLayer.cs

```diff
@@ -0,0 +1,421 @@
+using Esri.ArcGISMapsSDK.Components;
+using Esri.ArcGISMapsSDK.Utils.GeoCoord;
+using Esri.GameEngine.Geometry;
+using FeatureLayerData;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using UnityEngine.Networking;
+using UnityEngine;
+using UnityEngine.UI;
+
+[System.Serializable]
+public struct WebLink
+{
+    public string Link;
+    public string[] RequestHeaders;
+}
+
+[System.Serializable]
+public class Feature
+{
+    public Geometry geometry = new Geometry();
+    public Properties properties = new Properties();
+}
+
+[System.Serializable]
+public class Properties
+{
+    public List<string> propertyNames = new List<string>();
+    public List<string> data = new List<string>();
+}
+
+[System.Serializable]
+public class Geometry
+{
+    public List<double> coordinates = new List<double>();
+}
+
+public class FeatureLayer : MonoBehaviour
+{
+    [SerializeField] private ArcGISCameraComponent arcGISCamera;
+    [SerializeField] private Transform contentContainer;
+    private FeatureData featureInfo;
+    [SerializeField] private GameObject featurePrefab;
+    private int featureSRWKID = 4326;
+    private ArcGISLocationComponent locationComponent;
+    [SerializeField] private List<string> outfields = new List<string>();
+    private int stadiumSpawnHeight = 10000;
```

Comment:
```suggestion
    private float stadiumSpawnHeight = 10000.0f;
```
float makes more sense in this context